packaging/linux/openra.6

```diff
@@ -0,0 +1,68 @@
+.TH OPENRA 6
+.SH NAME
+openra \- An open-source implementation of the Command & Conquer: Red Alert engine using .NET/Mono and OpenGL.
+.SH SYNOPSIS
+.B openra
+[\fB\Game.Mod=\fR\fImodchooser\fR]
+.SH DESCRIPTION
+.B openra
+starts the game.
+.SH OPTIONS
```

Comment:
I thought about it already and rejected the idea. Most of the https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Game/Settings.cs are not very interesting and having them all exposed will add too much bloat. Also these settings are pretty stable. It is way quicker to do it by hand now. This can be done later in another pull request.